modules/avcodec/parse/h2645_parse.h

```diff
@@ -0,0 +1,40 @@
+#ifndef H2645_PARSE_H
+#define H2645_PARSE_H
+#include <stdint.h>
+
+/**
+ * split sps and pps
+ */
+int h264_get_sps_pps(uint8_t *data, int len,
+			uint8_t *sps, int *sps_len,
+			uint8_t *pps, int *pps_len);
+
+/**
+ * split sps and pps and sei
+ */
+int h264_get_sps_pps_sei(uint8_t *data, int len,
+			uint8_t *sps, int *sps_len, uint8_t *pps, int *pps_len,
+			uint8_t *sei, int *sei_len);

```

Comment:
is this function used in your PR ?

SEI is "Supplemental Enhancement Information" and contains info like timing and metadata.
Is SEI required to be set in the extradata ?